app/main/__init__.py

```diff
@@ -1,6 +1,9 @@
 from flask import Blueprint
+from .helpers.auth import requires_auth
 
 main = Blueprint('main', __name__)
 
+main.before_request(requires_auth)
```

Comment:
Can this be wrapped in an ``if not test`` type thing?  There is the Test config that we can use. Think it should work.